org.civicrm.reqangular/test/services/api_test.js

```diff
@@ -0,0 +1,122 @@
+define([
+  'common/angular',
+  'common/angularMocks',
+  'common/services/api'
+], function () {
+  'use strict';
+
+  describe('api test', function () {
+    var api, $httpBackend, $rootScope;
+
+    beforeEach(module('common.apis'));
+
+    beforeEach(inject(function (_api_, _$httpBackend_, _$rootScope_) {
+      api = _api_;
+      $rootScope = _$rootScope_;
+      $httpBackend = _$httpBackend_;
+    }));
+
+    describe('sendGET', function () {
+      var entity = 'send_get_entity',
+        action = 'send_get_action',
+        params = {key: 'value'},
+        cache = true,
+        promise;
+
+      afterEach(function () {
+        $httpBackend.flush();
```

Comment:
can you probably do this outside of any sub-`describe` given that you need to do it for all of the tests anyway